src/authorizations/components/TokenRow.tsx

```diff
@@ -59,7 +64,7 @@ class TokenRow extends PureComponent<Props> {
           {[
             <React.Fragment key={auth.id}>
               Created at:{' '}
-              {moment(auth.createdAt).format(UPDATED_AT_TIME_FORMAT)}
+              {updated}
```

Comment:
`updated` is kind of a weird name for this date, especially since it represents `created_at` which is often contrasted with `updated_at`

personally, i think it's fine to just do
```ts
{formatter.format(date)}
```